src/App.js

```diff
@@ -2,11 +2,10 @@ import logo from './logo.svg';
 import './App.css';
 
 function App() {
+  const API_KEY = process.env.local.REACT_APP_SPOTIFY_API_KEY;
```

Comment:
hmmm i dont know wether this is correct way to use the API key stored in the env.local files. 
i hope I not in the wrong side here... hehehhe

Reply:
hahaha

Reply:
hihihihih